less/drupal.less

```diff
@@ -21,13 +21,25 @@
 
 // Default components
 @import '_components/_mixin.less';
+//@import '_components/ads.less';
```

Comment:
Can we remove these commented out lines?

Reply:
I would like to keep them since it's useful to know what file we are not including in our drupal front-end stack.